external-crates/move/crates/move-vm-runtime/src/loader/chain_ast.rs

```diff
@@ -0,0 +1,29 @@
+// Copyright (c) The Move Contributors
+// SPDX-License-Identifier: Apache-2.0
+
+use crate::loader::ast::RuntimePackageId;
+use move_binary_format::CompiledModule;
+use move_core_types::language_storage::ModuleId;
+use std::collections::BTreeMap;
+
+#[derive(Debug, Clone)]
+pub(crate) struct DeserializedPackage {
```

Comment:
Consider `BinaryFormatPackage` or something, since it holds things from the `move_binary_format`?

Reply:
type table may be fine here, but linkage table would definitely not be correct here (IMO). This package is being loaded "irrespective" of the context. The context that the package is then linked in/interpreted in is a dynamic construct and IMO shouldn't reside on this most likely. But lets chat about it :)